crates/red_knot_python_semantic/resources/mdtest/stubs/ellipsis.md

```diff
@@ -0,0 +1,42 @@
+# Ellipsis
+
+## Function and Methods
+
+For default values the ellipsis literal `...` can be used.
+
+```py
+from typing import Dict
+
+def f(x: int = ...) -> None: ...
+def f2(x: Dict = ...) -> None: ...
+
+# error: [invalid-assignment] "Object of type `EllipsisType | ellipsis` is not assignable to `int`"
+y: int = ...
```

Comment:
This seems redundant with the dedicated "Class and Module Level Attributes" test below.
```suggestion
```